testsuite/features/support/constants.rb

```diff
@@ -196,166 +196,326 @@
+                            'Debian 10 (buster) pool for amd64 for Uyuni' => 'debian-10-pool-amd64-uyuni',
+                            'Debian 11 (bullseye) pool for amd64 for Uyuni' => 'debian-11-pool-amd64-uyuni',
+                            'openSUSE Leap 15.4 (aarch64)' => 'opensuse_leap15_4-aarch64',
+                            'openSUSE Leap 15.5 (aarch64)' => 'opensuse_leap15_5-aarch64'
+                          } }.freeze
 
 # Used for creating bootstrap repositories
+# The keys are the values of BASE_CHANNEL_BY_CLIENT
 # The values can be found out on the server by running 'mgr-create-bootstrap-repo'
 # Then select the correct name for the product you want
-CHANNEL_TO_SYNC_BY_BASE_CHANNEL = { 'SLE-Product-SUSE-Manager-Proxy-4.3-Pool' => 'SUMA-43-PROXY-x86_64',
-                                    'SLES12-SP4-Pool' => 'SLE-12-SP4-x86_64',
-                                    'SLES12-SP5-Pool' => 'SLE-12-SP5-x86_64',
-                                    'SLES15-SP1-Pool' => 'SLE-15-SP1-x86_64',
-                                    'SLES15-SP2-Pool' => 'SLE-15-SP2-x86_64',
-                                    'SLES15-SP3-Pool' => 'SLE-15-SP3-x86_64',
-                                    'SLES15-SP4-Pool' => 'SLE-15-SP4-x86_64',
-                                    'SLES15-SP5-Pool' => 'SLE-15-SP5-x86_64',
-                                    'SUSE-MicroOS-5.1-Pool for x86_64' => 'SLE-MICRO-5.1-x86_64',
-                                    'SUSE-MicroOS-5.2-Pool for x86_64' => 'SLE-MICRO-5.2-x86_64',
-                                    'SLE-Micro-5.3-Pool for x86_64' => 'SLE-MICRO-5.3-x86_64',
-                                    'SLE-Micro-5.4-Pool for x86_64' => 'SLE-MICRO-5.4-x86_64',
-                                    'almalinux9 for x86_64' => 'almalinux-9-x86_64',
-                                    'Fake Base Channel' => 'fake_base_channel-x86_64',
-                                    'RHEL x86_64 Server 7' => 'RES7-x86_64',
-                                    'EL9-Pool for x86_64' => 'SUSE-LibertyLinux9-x86_64',
-                                    'oraclelinux9 for x86_64' => 'oracle-9-x86_64',
-                                    'RHEL8-Pool for x86_64' => 'SLE-ES8-x86_64',
-                                    'rockylinux-9 for x86_64' => 'rockylinux-9-x86_64',
-                                    'ubuntu-2004-amd64-main' => 'ubuntu-20.04-amd64',
-                                    'ubuntu-2204-amd64-main' => 'ubuntu-22.04-amd64',
-                                    'debian-10-pool' => 'debian10-amd64',
-                                    'debian-11-pool' => 'debian11-amd64',
-                                    'openSUSE-Leap-15.4-Pool for aarch64' => 'openSUSE-Leap-15.4-aarch64',
-                                    'openSUSE-Leap-15.5-Pool for aarch64' => 'openSUSE-Leap-15.5-aarch64' }.freeze
+CHANNEL_TO_SYNC_BY_BASE_CHANNEL = { 'SUSE Manager' =>
```

Comment:
Channel label in fact.
But maybe this can be out of scope, as you keep the previous name.

Reply:
Yes. This would be something for another PR.